Assets/Scripts/Gameplay/NPC.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NPC : MonoBehaviour, IInteractable
+{
+    Animator animator;
+    [SerializeField] private string _prompt;
+    public string Prompt => _prompt;
```

Comment:
Check if you can remove the need for a serialized private string and only use the public property from IInteractable